.github/workflows/ohos.yml

```diff
@@ -240,3 +238,10 @@ jobs:
           [[ $servo_pid =~ ^[0-9]+$ ]] || { echo "It looks like servo crashed!" ; exit 1; }
           # If the grep fails, then the trace output for the "page loaded" prompt is missing
           grep 'org\.servo\.servo-.* tracing_mark_write.*PageLoadEndedPrompt' test_output/servo.ftrace
+      - name: "Run benchmark"
+        run: hitrace-bench --bencher -n 5
```

Comment:
Do we not need to pass the app name here? if it is optional, I would prefer to still explicitly give the app name and also the URL we are benching, just to make it more obvious to other people looking at this code.